beets/library.py

```diff
@@ -383,6 +398,8 @@ class Item(LibModel):
 
     _sorts = {'artist': SmartArtistSort}
 
+    _format_config_key = 'list_format_item'
```

Comment:
Let's add a docstring, either here or in `LibModel` (as with the series of docstrings for class variables in `beets.dbcore.db.Model`).